summary-visualiser/TODO.md

```
@@ -0,0 +1,14 @@
+# TODO
+
+## Ways to display data
+
+* are `within_?std` values useful, or can we just show `std`?
+* What max precision should we show?
+
+## Metrics that I don't know how to interpret
+
+* `dht_sync_lag`
+    * `sync_lag_rate` -- what is the unit?
+    * `cascade_duration` -- amount of time spent doing cacade queries?
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)